website/settings/defaults.py

```diff
@@ -1955,9 +1963,10 @@ class CeleryConfig:
 OOPSPAM_APIKEY = None
 OOPSPAM_SPAM_LEVEL = 3  # The minimum level (out of 6) that is flagged as spam.
 OOPSPAM_CHECK_IP = True  # Whether OOPSpam checks IP addresses. When testing locally, turn this off
+OOPSPAM_ENABLED = False
 
 # spam options
-SPAM_CHECK_ENABLED = False
+SPAM_SERVICES_ENABLED = False
```

Comment:
y tho? @mattclark looks like this is this going to need a config change on Prod to avoid breaking all of our Spam checking when the release goes out

Reply:
I wanted to be able to separate the network dependent code from the domain extraction for test purposes and general clarity.

Reply:
Can you link the release playbook, please? not seeing it with the rest in Notion